app/components/funds.js

```diff
@@ -56,7 +56,11 @@ const Funds = () => (
               <Link href="https://opencollective.com/babel" theme={Link.THEME_DARK}>
                 Babel
               </Link>{' '}
-              with $200/month
```

Comment:
you need remove the babel link, as we're not funding the project anymore, we switched from it to code for romania